src/Base/Support/Utils.php

```diff
@@ -41,32 +41,46 @@ public static function cast(string $class, ...$args)
     }
 
     /**
-     * @param  array       $array
-     * @param  string|null $case
+     * @param  array    $array
+     * @param  null|int $flags
      *
      * @return array
      */
-    public static function changeArrayKeysCase(array $array, string $case = null): array
+    public static function changeArrayKeysCase(array $array, ?int $flags = null): array
     {
         $newArray = [];
 
         foreach ($array as $key => $value) {
```

Comment:
```suggestion
        $newArray = $array;

        foreach ($newArray as $key => $value) {
```
Met de foreach zoals hij nu is pas je $array aan en volgens mij wil je dat niet. Je zou er een kopie van kunnen maken en die aanpassen zodat $array ongewijzigd blijft.

Reply:
Is niet nodig, `$array` wordt niet aangepast, alleen uitgelezen